docsets/Carbon/docset.json

```diff
@@ -1,14 +1,28 @@
 {
     "name": "Carbon",
-    "version": "1.30.0",
-    "archive": "Carbon-1.30.0.tgz",
+    "version": "1.31.0",
+    "archive": "versions/Carbon-1.31.0.tgz",
```

Comment:
This can't point to an archive within `specific_versions`. It needs to be an archive of its own, even if it's a duplicate.